jdbc/src/main/java/org/apache/iotdb/jdbc/ResultSetMetaDataAdapter.java

```diff
@@ -0,0 +1,103 @@
+package org.apache.iotdb.jdbc;
+
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+
+public abstract class ResultSetMetaDataAdapter implements ResultSetMetaData {
```

Comment:
Why use ResultSetMetaDataAdapter to wrap interface ResultSetMetaData for IoTDBMetadataResultMetadata and IoTDBResultMetadata?

Reply:
Maybe to elimate the duplicated code？

Reply:
These kine of modifications will be revoked.

Reply:
yes to eliminate the duplicated code, but seems to be needless since there are only two subclasses of this adapter.